network/shuffle/src/main/java/org/apache/spark/network/shuffle/ExternalShuffleBlockResolver.java

```diff
@@ -110,6 +110,9 @@ public ManagedBuffer getBlockData(String appId, String execId, String blockId) {
       return getHashBasedShuffleBlockData(executor, blockId);
     } else if ("org.apache.spark.shuffle.sort.SortShuffleManager".equals(executor.shuffleManager)) {
       return getSortBasedShuffleBlockData(executor, shuffleId, mapId, reduceId);
+    } else if ("org.apache.spark.shuffle.unsafe.UnsafeShuffleManager".equals(
```

Comment:
this could just be merged with the previous case:
```
} else if ("org.apache.spark.shuffle.sort.SortShuffleManager".equals(executor.shuffleManager) ||
    "org.apache.spark.shuffle.unsafe.UnsafeShuffleManager".equals(executor.shuffleManager)) {
  return getSortBasedShuffleBlockData(...);
}
```